Sources/SDK/CustomerIO.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+
+// disabling lint rule to allow 2 letter enum names: US, EU, etc.
+// swiftlint:disable identifier_name
+/**
+ Region that your Customer.io Workspace is located in.
+
+ The SDK will route traffic to the correct data center location depending on the `Region` that you use.
+ */
+public enum Region: String, Equatable {
+    /// The United States (US) data center
+    case US = "us"
```

Comment:
Great question. The enum must be string or Int so that we can save the Region object in the key/value storage of the device. I prefer string over int as it's easier to debug and less prone to errors adding/removing regions in the future. Even if you ever change the order of the `case` statements in the code in the future, using an Int would break the code. String is preferred for this reason. 

My thought process behind using the lowercase `us` and `eu` could be improved so I'll just remove that requirement.